test/tokend-transformer.js

```diff
@@ -139,7 +139,8 @@ describe('TokendTransformer', function() {
         $tokend: {
           type: 'kms',
           resource: '/v1/kms/decrypt',
-          ciphertext: 'gbbe'
+          ciphertext: 'gbbe',
+          region: 'eu-central-1'
```

Comment:
Can you split this into two tests, one that includes the region with the request and one that doesn't? The idea is that if we accidentally delete the newly added region handling code from `lib/transformers/tokend.js`, one of the tests should break.